ouroboros-network/src/Ouroboros/Network/TxSubmission/Outbound.hs

```diff
@@ -189,7 +189,7 @@ txSubmissionOutbound tracer maxUnacked TxSubmissionMempoolReader{..} =
               !unackedMap' = foldl' (flip Map.delete) unackedMap txids
               client'      = client unackedSeq unackedMap' lastIdx
 
-          -- Trace the transactions to be sent in the response.
-          traceWith tracer (TraceTxSubmissionOutboundSendMsgReplyTxs txs)
+              -- Trace the transactions to be sent in the response.
+              traceTxsSent = traceWith tracer (TraceTxSubmissionOutboundSendMsgReplyTxs txs)
```

Comment:
This requires a comment (here or near L195) saying that tracing itself will be executed after sending the reply.